README.md

```diff
@@ -4,6 +4,8 @@
 
 Build this extension by following [the usual procedure for Slicer extensions](https://slicer.readthedocs.io/en/latest/developer_guide/extensions.html#build-an-extension).
 
+![Screenshot](https://github.com/OpenwaterHealth/SlicerOpenLIFU/blob/266-Publish-to-Extension-Index/blob/main/Screenshots/1.png)
```

Comment:
Oh I see what you did here.

> * Before this is merged, the screenshot link needs to point to where the screenshot _would_ be once this is merged. @ebrahimebrahim is there a way to [locally reference the image path in this commit](https://github.com/OpenwaterHealth/SlicerOpenLIFU/pull/278/commits/6ab25b146228d6fae6b9c7d5961d068c4c3404be) so that the link can always be valid?

Yes you can just use a relative path and github markdown should figure it out. this blob/main thing is kind of weird, you can just create a screesnshots folder and add things to it. then use a relative path in the markdown